src/shogun/clustering/GMM.cpp

```diff
@@ -826,5 +835,5 @@ void CGMM::register_params()
 	//TODO serialization broken
 	//m_parameters->add((SGVector<CSGObject*>*) &m_components, "m_components", "Mixture components");
```

Comment:
Could you try to enable this? Should work with the new framework at least (not sg add though, but watch param)

Reply:
It seems to work :) however, GMM needs some unit-tests to ensure proper validation. I may try to add some in a new PR.